docs/pages/configuration/service.mdx

```diff
@@ -34,6 +34,9 @@ The `port` option is mandatory and expect an integer with a port number. The ser
 ### `containerPort`
 The `containerPort` option is optional and tells the service to forward the traffic from the service port (defined in `port`) to a different `containerPort` of a pod.
 
+### `name`
+The `name` option is optional and allows you to customize the name of the port. By default ports will be named "port-0", "port-1" etc.
+
 #### Default for `containerPort`
```

Comment:
Thanks for making that change! I just noticed something else. The `name` section should be under the `Default for containerPort` section because that belongs to `containerPort`.

Reply:
Sorry I didn't catch that earlier.

Reply:
My bad! I have re-arranged it and made the defaults explanation consistent with the other ones now.